src/processTree.ts

```diff
@@ -157,8 +155,8 @@ export function getProcesses(one: (pid: number, ppid: number, command: string, a
 			reject(err);
 		});
 
-		proc.stderr?.setEncoding('utf8');
-		proc.stderr?.on('data', data => {
+		proc.stderr.setEncoding('utf8');
+		proc.stderr.on('data', data => {
```

Comment:
Same as above. Don't remove `?` here.

Reply:
You could notice, that updated version of original file uses better typed `ChildProcessWithoutNullStreams` which guarantees that these streams are never null